### ISSUE TITLE
Ctrl+C doesn't work

### ISSUE THREAD

**Issue by Bomper** [Voyz/ibeam#225]
**Describe the bug**

Imagine that after starting the container, you realize you mistyped your password... oops! Cltr+C and try again. Except that doesn't work :)

**To Reproduce**
```
sudo docker run --env IBEAM_ACCOUNT=xxx --env IBEAM_PASSWORD=yyy -p 5000:5000 voyz/ibeam
2024-12-07 12:29:41,953|I| ############ Starting IBeam version 0.5.5 ############
2024-12-07 12:29:41,962|I| Secrets source: env
2024-12-07 12:29:41,965|I| Health server started at port=5001
2024-12-07 12:29:41,966|I| Configuration:
{'INPUTS_DIR': '/srv/inputs/', 'OUTPUTS_DIR': '/srv/outputs', 'GATEWAY_DIR': '/srv/clientportal.gw', 'CHROME_DRIVER_PATH': '/usr/bin/chromedriver', 'GATEWAY_STARTUP': 20, 'GATEWAY_PROCESS_MATCH': 'ibgroup.web.core.clientportal.gw.GatewayStart', 'MAINTENANCE_INTERVAL': 60, 'SPAWN_NEW_PROCESSES': False, 'LOG_LEVEL': 'INFO', 'LOG_TO_FILE': True, 'LOG_FORMAT': '%(asctime)s|%(levelname)-.1s| %(message)s', 'REQUEST_RETRIES': 2, 'REQUEST_TIMEOUT': 15, 'RESTART_FAILED_SESSIONS': True, 'RESTART_WAIT': 15, 'REAUTHENTICATE_WAIT': 15, 'HEALTH_SERVER_PORT': 5001, 'SECRETS_SOURCE': 'env', 'GCP_SECRETS_URL': None, 'START_ACTIVE': True, 'GATEWAY_BASE_URL': 'https://localhost:5000', 'ROUTE_AUTH': '/sso/Login?forwardTo=22&RL=1&ip2loc=on', 'ROUTE_VALIDATE': '/v1/portal/sso/validate', 'ROUTE_REAUTHENTICATE': '/v1/portal/iserver/reauthenticate?force=true', 'ROUTE_INITIALISE': '/v1/api/iserver/auth/ssodh/init', 'ROUTE_AUTH_STATUS': '/v1/api/iserver/auth/status', 'ROUTE_TICKLE': '/v1/api/tickle', 'ROUTE_LOGOUT': '/v1/api/logout', 'USER_NAME_EL': None, 'PASSWORD_EL': 'NAME@@password', 'SUBMIT_EL': 'CSS_SELECTOR@@.btn.btn-lg.btn-primary', 'ERROR_EL': None, 'SUCCESS_EL_TEXT': 'TAG_NAME@@Client login succeeds', 'OAUTH_TIMEOUT': 15, 'PAGE_LOAD_TIMEOUT': 15, 'ERROR_SCREENSHOTS': False, 'MAX_FAILED_AUTH': 5, 'MIN_PRESUBMIT_BUFFER': 5, 'MAX_PRESUBMIT_BUFFER': 30, 'MAX_IMMEDIATE_ATTEMPTS': 10, 'IBKEY_PROMO_EL_CLASS': 'CLASS_NAME@@ibkey-promo-skip', 'AUTHENTICATION_STRATEGY': 'B', 'MAX_STATUS_CHECK_RETRIES': 120, 'MAX_REAUTHENTICATE_RETRIES': 3, 'UI_SCALING': 1.0, 'TWO_FA_EL_ID': 'ID@@twofactbase', 'TWO_FA_NOTIFICATION_EL': 'CLASS_NAME@@login-step-notification', 'TWO_FA_INPUT_EL_ID': 'ID@@chlginput', 'TWO_FA_HANDLER': None, 'STRICT_TWO_FA_CODE': True, 'TWO_FA_SELECT_EL_ID': 'ID@@sf_select', 'TWO_FA_SELECT_TARGET': 'IB Key', 'CUSTOM_TWO_FA_HANDLER': 'custom_two_fa_handler.CustomTwoFaHandler'}
2024-12-07 12:29:41,967|I| Gateway not found, starting new one...
2024-12-07 12:29:41,968|I| Note that the Gateway log below may display "Open https://localhost:[PORT] to login" - ignore this command.
2024-12-07 12:29:41,968|I| Starting Gateway as Linux process with params: ['bash', 'bin/run.sh', 'root/conf.yaml']
running  
 runtime path : root:dist/ibgroup.web.core.iblink.router.clientportal.gw.jar:build/lib/runtime/*
 config file  : root/conf.yaml
2024-12-07 12:29:41,992|I| Gateway started with pids: [13]
2024-12-07 12:29:41,995|I| Cannot ping Gateway. Retrying for another 20 seconds
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by io.netty.util.internal.ReflectionUtil (file:/srv/clientportal.gw/build/lib/runtime/netty-common-4.1.15.Final.jar) to constructor java.nio.DirectByteBuffer(long,int)
WARNING: Please consider reporting this to the maintainers of io.netty.util.internal.ReflectionUtil
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
2024-12-07 12:29:42,997|I| Cannot ping Gateway. Retrying for another 19 seconds
 -> mount demo on /demo
Java Version: 11.0.24
****************************************************
version: a27ed42161ad96c53e715ca5c5e3e3fa4cff5262 Mon, 24 Apr 2023 15:41:53 -0400
****************************************************
This is the Client Portal Gateway
for any issues, please contact api@ibkr.com
and include a copy of your logs
****************************************************
https://www.interactivebrokers.com/api/doc.html
****************************************************
Open https://localhost:5000 to login
App demo is available after you login under: https://localhost:5000/demo#/
2024-12-07 12:29:44,736|I| Gateway connection established
2024-12-07 12:29:44,908|I| NO SESSION Status(running=True, session=False, connected=False, authenticated=False, competing=False, collision=False, session_id=None, server_name=None, server_version=None, expires=None)
2024-12-07 12:29:44,908|I| Authentication strategy: "B"
2024-12-07 12:29:44,909|I| No active sessions, logging in...
2024-12-07 12:29:44,909|I| Loading auth webpage at https://localhost:5000/sso/Login?forwardTo=22&RL=1&ip2loc=on
2024-12-07 12:29:54,349|I| Gateway auth webpage loaded
2024-12-07 12:29:54,350|I| Login attempt number 1
2024-12-07 12:29:59,986|I| Submitting the form
2024-12-07 12:30:01,269|E| Error displayed by the login webpage: Invalid username password combination
2024-12-07 12:30:01,288|W| Increased presubmit buffer to 10
2024-12-07 12:30:02,328|I| Login attempt number 2
2024-12-07 12:30:13,482|I| Submitting the form
2024-12-07 12:30:13,915|E| Error displayed by the login webpage: Invalid username password combination
2024-12-07 12:30:13,932|W| Increased presubmit buffer to 15
2024-12-07 12:30:14,966|I| Login attempt number 3
^C2024-12-07 12:30:30,704|I| Submitting the form
2024-12-07 12:30:31,100|E| Error displayed by the login webpage: Invalid username password combination
2024-12-07 12:30:31,114|W| Increased presubmit buffer to 20
2024-12-07 12:30:32,163|I| Login attempt number 4
^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C
2024-12-07 12:30:52,806|I| Submitting the form
2024-12-07 12:30:53,133|E| Error displayed by the login webpage: Invalid username password combination
2024-12-07 12:30:53,145|W| Increased presubmit buffer to 25
2024-12-07 12:30:54,175|I| Login attempt number 5
^C^C^C^C^Z
^X^C^C^C^C^C2024-12-07 12:31:19,815|I| Submitting the form
2024-12-07 12:31:20,209|E| Error displayed by the login webpage: Invalid username password combination
2024-12-07 12:31:20,222|W| The presubmit buffer set to maximum: 30
2024-12-07 12:31:20,259|C| ######## ATTENTION! ######## Maximum number of failed authentication attempts (IBEAM_MAX_FAILED_AUTH=5) reached. IBeam will shut down to prevent an account lock-out. It is recommended you attempt to authenticate manually in order to reset the counter. Read the execution logs and report issues at https://github.com/Voyz/ibeam/issues
2024-12-07 12:31:20,259|I| Cleaning up the resources. Display: <pyvirtualdisplay.display.Display object at 0x7fb9d0c4ae50> | Driver: <selenium.webdriver.chrome.webdriver.WebDriver (session="63fb017620ac5d3c425183a739ee89e4")>
2024-12-07 12:31:20,331|I| Logging in failed
2024-12-07 12:31:20,331|W| Shutting IBeam down due to critical error.
^C^C^C^C^C^C^C^C^C^C^C^C^C


^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C^C
```

**Environment**
IBeam version: the one that comes with Docker pull
OS: Ubuntu 18

**Comment by Voyz** [comment]
Hey @Bomper thanks for bringing this to my attention 👍 I'm sorry you got your account locked out because of it. 

I've fixed it on `0.5.6` please check it out and let me know if it works for you

**Comment by Bomper** [comment]
The fix works, thank you very much!
